backend/module/eCampCore/src/Module.php

```diff
@@ -30,6 +33,9 @@ public function onBootstrap(MvcEvent $e) {
         // Enable next line for Doctrine debug output
         // $em->getConfiguration()->setSQLLogger(new EchoSQLLogger());
 
+        Type::overrideType('date', DateUtcType::class);
+        Type::overrideType('datetime', DateTimeUtcType::class);
```

Comment:
Is this the correct place - or should it be moved to eCampLib?
Reason: it's used in eCamp\Lib\Entity\BaseEntity

Reply:
Moved it to Module.php of the eCampLib. Is this the right location?